### ISSUE TITLE
Install Ghost via tarball

### ISSUE THREAD

**Issue by kirrg001** [TryGhost/Ghost-CLI#59]
I've added a tarball link instead of the Ghost version, that worked pretty good.
So instead of `version`, i have added my Ghost tarball link.
https://github.com/TryGhost/Ghost-CLI/blob/master/lib/tasks/npm-download.js#L27

How about adding an option for it?
Background: i don't want to wait until a new release zip is available. I would like to have an easy way to test my Ghost branch with Ghost-CLI. 

If there is already an easy way, we close this issue 🌮 !

**Comment by kirrg001** [comment]
we need to ensure submodules get installed correctly

**Comment by ErisDS** [comment]
I think this steps outside the realms of what the ghost-cli is intended to do.

In order to install from a tarball, Ghost-cli would need to know about and be able to handle all of the build steps for Ghost, including handling the submodules, npm & bower install for ghost-admin, ember build, etc etc.

I agree that it would be useful to have a way to test ghost-cli, however I don't think that installing direct from a github tarball is the right solution. Instead, it would be better (I think) to perhaps be able to link a local pre-built zip or similar.

**Comment by kirrg001** [comment]
> Instead, it would be better (I think) to perhaps be able to link a local pre-built zip or similar.

Agree. Will rename the issue to something general. At the end: i am fine with any solution as long it's official and easy to use :)

**Comment by acburdine** [comment]
Personally I'm in favor of a `ghost link` command - you could pass it the path to your local ghost install and it would symlink it into the ghost-cli setup.

**Comment by kirrg001** [comment]
> Personally I'm in favor of a ghost link command - you could pass it the path to your local ghost install and it would symlink it into the ghost-cli setup.

👍

**Comment by cbanowsky** [comment]
Just as nvm has the `nvm ls-remote' Ghost should have a `ls` ans `ls-remote` so specific releases can be installed and then choose an unstable version if they want to run bleeding edge.

**Comment by acburdine** [comment]
@cbanowsky If you're talking about installing the master branch of Ghost, then I'd say no, as doing so requires several other commands needed to set up development dependencies and build the ember admin application, all of which is outside of Ghost-CLI's scope IMO.

If you're talking about future (post-1.0) builds (such as nightly builds, and iirc they are an idea that's still on the table), then I'd say its not a bad idea to consider once those actually become a thing.
